components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -106,10 +106,13 @@ export class GithubApp {
                 return;
             }
             if (action === "renamed") {
-                const project = await this.projectDB.findProjectByInstallationId(String(installation.id))
-                if (project) {
-                    project.cloneUrl = repository.clone_url;
-                    await this.projectDB.storeProject(project);
+                const oldName = (ctx.payload as any)?.changes?.repository?.name?.from;
```

Comment:
This is undocumented stuff, I'm afraid. The event payload contains:
```
"changes": { "repository": { "name": { "from": "test-repo-123" } } }
```

To implement this in a robust way, we'd need to store `repository.id` to the `project`. next to the `cloneUrl`.

Reply:
makes sense. maybe put this as a TODO into the code